components/Button.js

```diff
@@ -9,10 +9,17 @@ const StyledButton = styled.button`
   filter: drop-shadow(5px 4px 4px #000000);
   margin: 0 1em;
   background-color: var(--background-primary);
-  border-radius: 5px;
+  color: var(--text-primary);
   cursor: pointer;
+  text-decoration: none;
```

Comment:
Ich denke man braucht text-decoration: None nur bei unordere List items oder? check das mal bitte, vielleicht geht es auch ohne

Reply:
hey, leider nicht. text-decoration none entfernt in diesem case, dass der text underlined ist.